superweather/superweather/LocationTool.h

```diff
@@ -0,0 +1,21 @@
+//
+//  LocationTool.h
+//  superweather
+//
+//  Created by mikael petersson on 2016-04-17.
+//  Copyright © 2016 mikael petersson. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+@interface LocationTool : NSObject<CLLocationManagerDelegate>
```

Comment:
I forgot about that. I originally intended to keep track of continuous changes to the location, but then I thought that it's probably not necessary.